osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -477,7 +477,7 @@ private static float offsetX(float dist, float halfHeight)
         {
             // The radius of the circle the carousel moves on.
             const float circle_radius = 3;
-            double discriminant = Math.Max(0, circle_radius * circle_radius - dist * dist);

```

Comment:
Yeah, all of these * before + ones are hurting readability more than they help. Not happy to merge all of those. :(